root/static/scripts/common/components/EntityLink.js

```diff
@@ -196,6 +197,17 @@ $ReadOnlyArray<Expand2ReactOutput> | Expand2ReactOutput | null => {
   let showDisambiguation = passedShowDisambiguation;
   let showIcon = passedShowIcon;
 
+  if (nameVariation === undefined && (
+    content !== undefined && (typeof content !== 'string')
+  )) {
+    const errorMessage = `Content of type ${typeof content} cannot be
+      compared as a string to entity name for name variation.`;
```

Comment:
minor, but to remove newlines and whitespace from the message (for translated strings that's handled by `cleanMsgid`)
```suggestion
    const errorMessage = `Content of type ${typeof content} cannot be ` +
      'compared as a string to entity name for name variation.';
```

Reply:
Good catch, now fixed: https://github.com/metabrainz/musicbrainz-server/compare/3e61be228c18742eae6d01be0fa504a8acc8d9f8..1a60e1fb39b41dc4088539feb5486151e10b427a